testing/unit_tests/test_loader_multi.py

```diff
@@ -0,0 +1,110 @@
+import os
+import shutil
+
+from ivadomed.loader.bids_dataframe import BidsDataframe
+from testing.unit_tests.t_utils import path_temp, download_multi_data
+from testing.common_testing_util import remove_tmp_dir, path_data_multi_sessions_contrasts_source, \
+    assert_empty_bids_dataframe, bids_dataframe_comparison_framework, path_data_multi_sessions_contrasts_tmp
+from pytest_cases import parametrize_with_cases
+import pytest
+from testing.unit_tests.cases_multi_sessions import *
+from testing.unit_tests.cases_multi_contrasts import *
+from testing.unit_tests.cases_multi_ground_truth import *
+
+
+def setup_function():
+    # Dedicated setup function for multi-session data.
+    remove_tmp_dir()
+    os.mkdir(path_temp)
+    if os.path.exists(path_data_multi_sessions_contrasts_source):
+        shutil.copytree(path_data_multi_sessions_contrasts_source,
+                        path_data_multi_sessions_contrasts_tmp,
+                        ignore=shutil.ignore_patterns(str(path_data_multi_sessions_contrasts_source / '.git')))
```

Comment:
Maybe we can move this part to `t_utils.py` just in case we may need this again in the future.

Reply:
Great point. Refactored for reuse and to be more clear on its purpose.